### DISCUSSION TITLE
中央大學授權軟體使用

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#346] [Ideas]
中央大學有提供許多軟體服務給學生與老師。因為我也沒有使用過，所以過程可能有問題。現在，我要一步一步帶各位安裝軟體。目前計畫帶各位安裝的軟體是LabVIEW與MATLAB。
安裝條件
 - 中央大學portal帳號
# 第一步
登入中央大學portal
# 第二步
校務行政->電算中心服務-授權軟體
![image](https://user-images.githubusercontent.com/86505454/149741665-c1133893-7cf9-4a23-9bc8-d187c687e5ab.png)
# 第三步
用滑鼠左鍵/觸控板左鍵點選「1.常用的電腦應用程式」
![image](https://user-images.githubusercontent.com/86505454/149742139-36f6de13-5f86-4103-a7e4-5ca6ccda038c.png)
# 第四步
用中央大學portal帳號登入
# 第五步
用滑鼠左鍵/觸控板左鍵點選「4.應用軟體」
![image](https://user-images.githubusercontent.com/86505454/149742304-4b57cce6-c856-4a5a-af93-fb05bd8d28c6.png)
# 第六步
看到LabVIEW與MATLAB
![image](https://user-images.githubusercontent.com/86505454/149742411-045c9dd1-c0ec-49cc-9d06-c18ae0ddcd2a.png)

**Comment by JunTingLin** [comment]
office系列產品也可以透過[校園授權軟體](https://ca.ncu.edu.tw/software/)下載喔!
在透過學校KMS認證就可以~~無限暢飲~~使用啦
詳細可以在參考這篇 [🧨我要~~榨乾~~學校(office、Windows通通免費) #390 ](https://github.com/NCU-GS4719-Python/Python-Community/discussions/390)

**Comment by Ben6uo** [comment]
哇!真假?學校竟然有那麼好的東西給我們 還是免費的!

**Reply by gnsJhenJie** [reply]
買軟體的經費有些是從學雜費裡面的電腦及網路使用費支出，所以嚴格來說也不算是免費XD

**Comment by chupochun** [comment]
謝謝這位大佬 一次分享這麼多福利給我們!!!

**Comment by pekoranoinu** [comment]
真是謝謝大佬分享這麼好的東西~

**Comment by xkllkx** [comment]
謝謝分享，光office直接省了一大筆。

**Comment by lx3211** [comment]
我用macOS也是載學校授權的office
不然只是為了開檔案還要花錢買實在太浪費了

**Comment by hapiwangy** [comment]
學校可以提供這種服務真的是太方便了八XD，我也要來榨乾學校的資源

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來學校有提供這些軟體。

**Comment by yucheng1028** [comment]
太水了拉，雖然我已經用了

**Comment by PoooYen** [comment]
大學其實資源很多，只是大多人都只知道繳學費不知道有東西可以用，感謝分享，也希望學校能多多宣傳學校的既有資源

**Comment by Yuhsin000** [comment]
錢繳了就是要用好用滿，原來在這裡安裝!!!

**Comment by SamTU0222** [comment]
謝謝分享！尤其學校還有提供office，這些也很實用！

**Comment by minniewei** [comment]
感謝分享,意外發現學校其實也提供非常多的資源

**Comment by HaoFish123** [comment]
謝謝分享，Office的資源真的讚。

**Comment by sys511613** [comment]
雖然windows跟office每半年要重新認證一次
但免費的東西就是真香

**Comment by bloodtrail0511** [comment]
那我之前都白買了..挖勒

**Comment by Yao900108** [comment]
使用這些真的可以省一筆不小的錢

**Comment by awkward-willy** [comment]
謝謝分享
先前逛的時候發現有許多實用的內容
若能妥善運用真的能精進自己同時省下一筆不小的開銷
